srcpkgs/please/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'please'
+pkgname=please
+version=0.3.16
+revision=1
+_commit="d330c5b24018ad6144eeb12f382ca5c45c885640"
+wrksrc="${pkgname}-${_commit}-${_commit}"
+build_style=cargo
+makedepends="cmake llvm clang pam-devel"
+depends="pam"
+short_desc="Please, a polite and easy to configure regex-first sudo alternative "
+maintainer="Ed Neville <ed@s5h.net>"
+license="GPL-3.0-or-later"
+homepage="https://www.usenix.org.uk/content/please.html"
+distfiles="https://gitlab.com/edneville/please/repository/archive.tar.gz?ref=$_commit>archive.tar.gz"
+checksum=8ab2db68a20e4dc2b2bfd1e7f46d2dd3cef53e7db1cf0e120e6add3043a00a5b
+
+pre_build() {
+	export CFLAGS_${RUST_BUILD//-/_}="${CFLAGS_host}"
+	export CC_${RUST_BUILD//-/_}="${BUILD_CC}"
+}
+
+post_install() {
+	local manpage
+	vdoc README.md
+	manpage=$(find ${wrksrc} -name please.ini.5 )
+	vman ${manpage}
+	manpage=$(find ${wrksrc} -name please.1 )
+	vman ${manpage}
+	please=$(find ${wrksrc} -name please )
+	pleaseedit=$(find ${wrksrc} -name pleaseedit )
```

Comment:
Aren't these covered by `do_install`?

You could just `chmod` them in `$DESTDIR/usr/bin`.